src/nimblepkg/cli.nim

```diff
@@ -13,6 +13,7 @@
 #   - Normal for MediumPriority.
 
 import logging, terminal, sets, strutils, os
+import ./version
```

Comment:
Why is this required? Should it be "common"?

Reply:
I just grepped for the definition of `NimbleError` and found it in version :)  I'll change it to `common`